src/Serialization.purs

```diff
@@ -163,11 +171,7 @@ convertTxOutput (T.TransactionOutput { address, amount, data_hash }) = do
     T.StakeCredentialScript (T.ScriptHash scriptHash) -> do
```

Comment:
Not really deserialisation but minor stylistic nitpick - the `do` is unnecessary for these oneliner binds. Entirely optional to remove of course :)

Reply:
I prefer to leave `do` in case a new statement gets added in the future - then the diff is shorter. Plus, it's an easy to spot indicator that the result is monadic.